docs/SsoOn2Fa.md

```diff
@@ -1,37 +1,67 @@
 # SSO on Second Factor Authentications
-When the feature is enabled for the institution, when using a MFA token we create a cookie containing information about the token that was used during MFA.
-On every successive authentication presented to the Stepup-Gateway we check if the cookie contains a valid authentication for the specified LoA.
-If the cookie meets the requirements, the authenticating user is not asked for the second factor authentication.
-When the service requires MFA, this can still be forced regardless of the SSO cookie, by adding the ForceAuthn attribute on the AuthnRequest element.
 
-## The cookie
-The cookie contains several values, used to ascertain if SSO can be given. These values are:
+In Stepup-Gateway 4.2.0 the possibility to have single-sign-on (SSO) on the second factor was introduced. SSO works by storing a cookie in the user's browser after they have successfully with a second factor. Then, on a successive authentication that requires 2FA, when a valid SSO cookie is presented and several other policy checks are passed, the user is not asked for the second factor and is authenticated immediately. You can see the SSO cookie as a temporary second factor bearer token 
```

Comment:
Typo in sentence:

| SSO works by storing a cookie in the user's browser after **they have successfully with** a second factor

Add 'authenticated' after successfully?